ddtrace/tracer/tracer_go11.go

```diff
@@ -0,0 +1,16 @@
+// +build go1.11
+
+package tracer
+
+import (
+	"context"
+	t "runtime/trace"
+)
+
+func startExecutionTracerTask(name string) func() {
```

Comment:
The name seems a bit long, and `ExecutionTracer` doesn't mean much to me unless I already spent time in `runtime/trace`.